README.md

```diff
@@ -1,53 +1,62 @@
 # 🧰 Fabulously Optimized · Vanilla Installer
 
-### Abbreviations
+<a href="https://gitmoji.dev">
+  <img src="https://img.shields.io/badge/gitmoji-%20😜%20😍-FFDD67.svg?style=flat-square" alt="Uses Gitmoji">
+</a>
+
+[![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
+
+## Abbreviations
 
 **FO** - Fabulously Optimized
 
 **VI** - Vanilla Installer
 
 ## Installation
 
-You heard that right, you need install the installer.
+You heard that right, you need to install the installer.
 
 ### Binaries (for newbies; easy to install)
 
 These are quite simple to install.
-Just [head to the **Releases** tab](releases) and download the correct file. 
+Just [head to the **Releases** tab](https://github.com/Fabulously-Optimized/vanilla-installer/releases/latest) and download the correct file.
 
 - Pros
-    - Easy to install
+  - Easy to install
 
 - Cons
-    - Mostly (more or less) stable versions are released, so you might have to wait a bit for newer features
+  - Mostly (more or less) stable versions are released, so you might have to wait a bit for newer features
 
 ### Scripts (for advanced users, still reasonably simple)
 
-**[Windows (outdated)](install/windows.bat) || [MacOS](install/macos.sh) || [Linux](install/linux.sh)**
+**[Windows](install/windows.bat) || [MacOS](install/macos.sh) || [Linux](install/linux.sh)**
 
 - Pros
-    - Can always used with the newest commits
+  - Can always used with the newest commits
```

Comment:
`Can always be used with the newest commits`

Since this part has a typo.

Reply:
Can't believe I didn't catch that lol